pkg/flare/archive.go

```diff
@@ -550,6 +555,30 @@ func zipRegistryJSON(tempDir, hostname string) error {
 	return err
 }
 
+func zipVersionHistory(tempDir, hostname string) error {
+	originalPath := filepath.Join(config.Datadog.GetString("logs_config.run_path"), "version-history.json")
```

Comment:
Maybe handle the loading of `logs_config.run_path` separately, as it may not be configured/something might go wrong when loading it?

Reply:
config.BindEnvAndSetDefault("run_path", defaultRunPath)